alchemist-project.el

```diff
@@ -135,7 +135,41 @@ in a new window."
          (filename (format "%s/%s" (alchemist-project-root) filename)))
     (if (file-exists-p filename)
         (find-file-other-window filename)
-      (message "No test file found."))))
+      (if (y-or-n-p "No test file found; create one now?")
+          (alchemist-project--create-test-for-current-file
+           filename (current-buffer))
+        (message "No test file found.")))))
+
+(defun alchemist-project--create-test-for-current-file (filename buffer)
+  "Creates and populates a test module, FILENAME, for the code in BUFFER.
+The module name given to the test module is determined from the name of the
+first module defined in BUFFER."
+  (let* ((directory-name (file-name-directory filename))
+         (module-name (alchemist-project--grok-module-name buffer))
+         (test-module-name (concat module-name ".Test")))
```

Comment:
Could remove the `.` here, because the Elixir convention is to use `ModuleTest` and not `Module.Test`.